app/jobs/verify_identity_with_alloy_job.rb

```diff
@@ -0,0 +1,10 @@
+class VerifyIdentityWithAlloyJob < ApplicationJob
+  queue_as :default
```

Comment:
I change the `queue_as` here.  It was `:validations`, but that caused it not to run in my local tests.  Maybe we need to do something with our seed logic to add the `validations` queue to Sidekiq?

Reply:
you need to expressly start the queue. I think I updated the foreman file to do that?

Reply:
@MatthewSchultz Maybe it never got committed? https://github.com/usdigitalresponse/papua-id-demo/blob/master/.foreman